data-es2016plus.js

```diff
@@ -81,7 +73,6 @@ exports.tests = [
           typescript: true,
           edge13: flag,
           edge14: true,
-          firefox48: firefox.nightly,
```

Comment:
This is implemented and is enabled by default only on nightly builds.  I believe the correct here would be to change firefox48 to "flag" and add a line with firefox52: nightly.